sdk/maps/azure-maps-search/src/test/java/com/azure/maps/search/TestUtils.java

```diff
@@ -0,0 +1,214 @@
+package com.azure.maps.search;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.models.GeoLineString;
+import com.azure.core.models.GeoObject;
+import com.azure.core.test.TestBase;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+import com.azure.core.util.serializer.TypeReference;
+import com.azure.maps.search.implementation.helpers.Utility;
+import com.azure.maps.search.implementation.models.PolygonPrivate;
+import com.azure.maps.search.implementation.models.ReverseSearchAddressResultPrivate;
+import com.azure.maps.search.implementation.models.ReverseSearchCrossStreetAddressResultPrivate;
+import com.azure.maps.search.implementation.models.SearchAddressResultPrivate;
+import com.azure.maps.search.models.BatchReverseSearchResult;
+import com.azure.maps.search.models.BatchSearchResult;
+import com.azure.maps.search.models.PointOfInterestCategoryTreeResult;
+import com.azure.maps.search.models.Polygon;
+import com.azure.maps.search.models.ReverseSearchAddressResult;
+import com.azure.maps.search.models.ReverseSearchCrossStreetAddressResult;
+import com.azure.maps.search.models.SearchAddressResult;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+
+import org.junit.jupiter.params.provider.Arguments;
+
+
+public class TestUtils {
+
+    static final String FAKE_API_KEY = "1234567890";
+    public static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(30);
+
+    static Polygon getPolygon(File file) throws IOException {
+        SerializerAdapter jacksonAdapter = JacksonAdapter.createDefaultSerializerAdapter();
+        TypeReference<PolygonPrivate> interimType = new TypeReference<PolygonPrivate>(){};
+        byte[] data = Files.readAllBytes(file.toPath());
+        PolygonPrivate polygonPrivate = jacksonAdapter.<PolygonPrivate>deserialize(data, interimType.getJavaType(),
+           SerializerEncoding.JSON);
+        Polygon polygon = Utility.toPolygon(polygonPrivate);
+        return polygon;
+    }
+
+    static List<Polygon> getMultiPolygonsResults() throws StreamReadException, DatabindException, IOException {
+        List<Polygon> result = new ArrayList<>();
+        File file1 = new File("src/main/resources/polygon1.json");
```

Comment:
Can't do this. This is probably the major change in this PR. This will only work if the test is being executed from inside azure-maps-search module. if it's being executed from mvn install from the root, it will not work.

Look into doing this: InputStream is = MyTest.class.getResourceAsStream("/test.csv"); This will read the file wherever it is in the classpath.

Reply:
This is still dependent on a specific path. Please look at how to fetch files using getResourceAsStream(). It's how MapsCommon does it as well.